tests/test_other.py

```diff
@@ -7094,7 +7094,7 @@ def test_metadce_minimal(self, *args):
 
   @node_pthreads
   def test_metadce_minimal_pthreads(self):
-    self.run_metadce_test('minimal.c', ['-Oz', '-sUSE_PTHREADS', '-sPROXY_TO_PTHREAD'], [], [])
+    self.run_metadce_test('minimal.c', ['-Oz', '-sUSE_PTHREADS'], [], [])
```

Comment:
But not there is no actual usage of threading in this example.. so I don't think its testing what we want it to test anymore.

Perhaps instead create a `minimal_main.c` which does have main.  Or alternatively create a `minimal_threads.c` which creates at least one thread.

Reply:
Makes sense, done.